docs/getting-started/dashboard_tutorial.md

```diff
@@ -0,0 +1,1788 @@
+
+Let's look at the example below. Here we have a function called `index` that returns a `text` component (an in-built Reflex UI component) that displays the text "Hello World!".
+
+Next we define our app using `app = rx.App()` and add the component we just defined (`index`) to a page using `app.add_page(index)`. The definition of the app and adding a component to a page are required for every Reflex app.
+
+```python
+import reflex as rx
+
+
+def index() -> rx.Component:
+    return rx.text("Hello World!")
+
+app = rx.App()
+app.add_page(index)
+```
+
+This code will render a page with the text "Hello World!" when you run your app like below:
+
+```python eval
+rx.text("Hello World!")
+```
+
+```md alert info
+For the rest of the tutorial the `app=rx.App()` and `app.add_page` will be implied and not shown in the code snippets.
+```
+
+### Creating a table
+
+Let's create a new component that will render a table. We will use the `table` component to do this. The `table` component has a `root`, which takes in a `header` and a `body`, which in turn take in `row` components. The `row` component takes in `cell` components which are the actual data that will be displayed in the table.
+
+```python eval
+rx.table.root(
+        rx.table.header(
+            rx.table.row(
+                rx.table.column_header_cell("Name"),
+                rx.table.column_header_cell("Email"),
+                rx.table.column_header_cell("Gender"),
+            ),
+        ),
+        rx.table.body(
+            rx.table.row(
+                rx.table.cell("Danilo Sousa"),
+                rx.table.cell("danilo@example.com"),
+                rx.table.cell("Male"),
+            ),
+            rx.table.row(
+                rx.table.cell("Zahra Ambessa"),
+                rx.table.cell("zahra@example.com"),
+                rx.table.cell("Female"),
+            ),
+        ),
+    )
+```
+
+```python
+def index() -> rx.Component:
+    return rx.table.root(
+        rx.table.header(
+            rx.table.row(
+                rx.table.column_header_cell("Name"),
+                rx.table.column_header_cell("Email"),
+                rx.table.column_header_cell("Gender"),
+            ),
+        ),
+        rx.table.body(
+            rx.table.row(
+                rx.table.cell("Danilo Sousa"),
+                rx.table.cell("danilo@example.com"),
+                rx.table.cell("Male"),
+            ),
+            rx.table.row(
+                rx.table.cell("Zahra Ambessa"),
+                rx.table.cell("zahra@example.com"),
+                rx.table.cell("Female"),
+            ),
+        ),
+    )
+```
+
+Components in Reflex have `props`, which can be used to customize the component and are passed in as keyword arguments to the component function. 
+
+The `rx.table.root` component has for example the `variant` and `size` props, which customize the table as seen below.
```

Comment:
Size makes sense, maybe as a reader I don't understand what variant is or what values it can be. I don't know though, maybe I naturally try to understand too much on the first read.

Reply:
I think it is okay, it is something they will need to learn so I think it gives a nice simple intro to it

Reply:
Sure